docs/structured-streaming-programming-guide.md

```diff
@@ -2190,34 +2190,34 @@ The function offers a simple way to express your processing logic but does not a
 deduplicate generated data when failures cause reprocessing of some input data. 
 For that situation you must specify the processing logic in an object.
 
-1. The function takes a row as input.
+- First, the function takes a row as input.
```

Comment:
The following is the current screenshot of our 2.4.3 page. The enumeration doesn't work well with template. Also, the python example has redundant leading spaces inconsistently from the other Python examples.
![Screen Shot 2019-08-02 at 10 48 02 PM](https://user-images.githubusercontent.com/9700541/62408087-9f4efb80-b577-11e9-8305-d97255108e1f.png)